src/main/java/co/rsk/federate/watcher/FederationWatcherListenerImpl.java

```diff
@@ -39,6 +39,30 @@ public void onRetiringFederationChange(Federation newRetiringFederation) {
         triggerClientChange(btcToRskClientRetiring, newRetiringFederation);
     }
 
+    @Override
+    public void onProposedFederationChange(Federation newProposedFederation) {
+        if (newProposedFederation == null) {
+            logger.info(
+                "[onProposedFederationChange] New proposed federation changed to none");
```

Comment:
```suggestion
                "[onProposedFederationChange] Proposed federation was cleared.");
```

something like that? wdyt?

Reply:
Updated